src/components/WorldListPage/WorldListPage.tsx

```diff
@@ -24,6 +24,7 @@ import LoggedInDetailPage from 'components/LoggedInDetailPage'
 import { NavigationTab } from 'components/Navigation/Navigation.types'
 import { Props, SortBy } from './WorldListPage.types'
 import './WorldListPage.css'
+import { ChainId, getChainName } from '@dcl/schemas'
```

Comment:
Move dependency from node_modules to the top of the other local dependencies.

Underneath decentraland-ui